### ISSUE TITLE
Delete all history

### ISSUE THREAD

**Issue by turbobuilt** [lmstudio-ai/lmstudio-bug-tracker#80]
Hi,

There's no easy way to delete all history. Should be relatively simple to add a button to clear all history (may want to add last hour, etc.)

**Comment by europeanplaice** [comment]
You can delete all conversation history following these steps.
1. Navigate to the directory where the conversation history is stored by clicking the ellipsis button in the chat history and selecting "Show in File Explorer".
2. Delete all .json files in the folder to remove the conversation history.

**Comment by dtsoden** [comment]
I agree with both @turbobuilt  and with @europeanplaice  - BOTH methods are needed and should/can be used. Why? because LM Studio supports 3 user personas.

1. User
2. Power User
3. Developer

@turbobuilt ask is germane for Persona #1 and @europeanplaice response is applicable to personas #2 & #3 

If this feature is not to be considered, then consider removing persona #1 so this tool is used by power users plus.

**Comment by chris-hatton** [comment]
This feature is badly needed in the UI.
